cmd/help.go

```diff
@@ -315,7 +317,8 @@ func showBackend(name string) {
 			optionsType = "advanced"
 			continue
 		}
-		fmt.Printf("### %s options\n\n", strings.Title(optionsType))
+		optionsType = cases.Title(language.Und, cases.NoLower).String(optionsType)
```

Comment:
This is really ugly! the strings are always ASCII strings for which `strings.Title` works properly.

Reply:
It's not "neat", I can agree with that. Still, ascii or not, `strings.Title` is listed as deprecated on https://pkg.go.dev/strings#Title, so I would think it is better not to use it.

Reply:
OK. Let's go with the flow!